src/styles/blocks/star.css

```diff
@@ -0,0 +1,39 @@
+.stars-Wrapper {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
```

Comment:
we can use margins for block

Reply:
I don't understand what did you mean. Why can't I use flex?